index.js

```diff
@@ -29,11 +29,6 @@ function fail (err) {
 }
 
 function openConfig (cb) {
-  var userHome = require('user-home');
```

Comment:
Moved this logic because `user-home` was being called by `config-path` so even if the module returns `null` it wouldn't get to this test.